apps/playground-vite/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "scripts": {
+    "dev": "pnpm run server",
+    "prod": "pnpm run build && pnpm run server:prod",
+    "build": "vite build",
+    "server": "tsx ./server",
+    "server:prod": "cross-env NODE_ENV=production tsx ./server"
+  },
+  "dependencies": {
+    "@vitejs/plugin-react": "^2.2.0",
+    "body-parser": "^1.20.1",
+    "cookie-parser": "^1.4.6",
+    "cross-env": "^7.0.3",
+    "dotenv": "^10.0.0",
+    "express": "^4.18.1",
+    "next-auth": "^4.16.4",
+    "node-fetch-native": "^0.1.8",
```

Comment:
could we use `undici` it's shipped with Node 18+ and the most spec compliant AFAIK

Reply:
Having problems with `@babel/cli` with Vite 3 currently

```bash
@babel/cli doesn't appear to be written in CJS, but also doesn't appear to be a valid ES module (i.e. it doesn't have "type": "module" or an .mjs extension for the entry point). Please contact the package author to fix.
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

ReferenceError: exports is not defined
    at eval (/@fs/Users/me/next-auth/packages/next-auth/next/index.js:5:23)
```

Reply:
or let's not make it part of the workspace for now and just pin the next-auth version